src/build/render/features/keywords.js

```diff
@@ -0,0 +1,10 @@
+const R = require("ramda");
+
+module.exports = async function({page, lang}) {
+  return {
+    keywords: [
+      page.tryLocalizedSlug(lang),
+      ...R.pathOr([], ["keywords", lang], page)
+    ]
+  };
+};
```

Comment:
This is a very simple example of a render feature. It takes the render context, pulls the language-specific keywords out of the page's metadata, and returns those in the `keywords` array so they're included in the search index. Other render features can specify their own keywords from entirely different sources (e.g. the tags feature adds the tag engine ID as a search keyword for the page).